src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -198,8 +197,15 @@ public TimeSpan NonceLifetime
         /// <summary>
         /// Gets or sets a value indicating if a 'sub' claim is required.
         /// </summary>
-        [DefaultValue(false)]
-        public bool RequireSub { get; set; }
+        [DefaultValue(true)]
+        public bool RequireSub { get; set; } = RequireSubByDefault;
```

Comment:
According to the spec, the subject is required. There are some cases that might not provide the subject, and that is why we give the user the option to change RequireSubByDefault to false. @brentschmaltz , can you confirm this?